src/login/logind-action.c

```diff
@@ -366,6 +415,7 @@ static const char* const handle_action_verb_table[_HANDLE_ACTION_MAX] = {
         [HANDLE_SLEEP]                  = "sleep",
         [HANDLE_FACTORY_RESET]          = "perform a factory reset",
         [HANDLE_LOCK]                   = "be locked",
+        [HANDLE_SECURE_ATTENTION_KEY]   = "secure attention key pressed",
```

Comment:
this should be a verb in infinitive form, and describe the action to be executed, not the event that caused this

Reply:
Seeing "The system will suspend and later hibernate now!" as an example, I think "handle the secure attention key" fits